spark/src/main/java/org/opensearch/sql/spark/flint/operation/FlintIndexOpCancel.java

```diff
@@ -28,7 +27,7 @@ public FlintIndexOpCancel(
   }
 
   public boolean validate(FlintIndexState state) {
-    return state == FlintIndexState.REFRESHING || state == FlintIndexState.CANCELLING;
+    return state == FlintIndexState.REFRESHING || state == FlintIndexState.ACTIVE;
```

Comment:
@penghuo  can we review this? why no `active` ?
Should we keep cancelling in this as it is a transitioning state and a job is already running to cancel?

Reply:
After discussing we have concluded that only when index state is in REFRESHING, the jobId in flint index metadata is correct. So, REFRESHING is the only viable initial state to cancel the operation.